app/src/androidTest/java/com/android/solvit/shared/ui/map/GetDirectionsBubbleTest.kt

```diff
@@ -0,0 +1,75 @@
+package com.android.solvit.shared.ui.map
+
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.test.core.app.ApplicationProvider
+import com.android.solvit.shared.model.map.Location
+import org.junit.Rule
+import org.junit.Test
+
+class GetDirectionsBubbleTest {
+
+  @get:Rule val composeTestRule = createComposeRule()
+
+  @Test
+  fun testGetDirectionsBubble_DisplaysLocationName() {
+    val location = Location(name = "Test Location", latitude = 0.0, longitude = 0.0)
+    composeTestRule.setContent { GetDirectionsBubble(location = location, onDismiss = {}) }
```

Comment:
Nit: These two lines could be refactored in a setup function used in all your tests.